docs/user-guide/src/capm3/automated_cleaning.md

```diff
@@ -0,0 +1,35 @@
+# Automated Cleaning
+
+Before reading this page, please see [Baremetal Operator Automated Cleaning](../bmo/automated_cleaning.md) page.
+
+If you are using only Metal3 Baremetal Operator, you can skip this page and refer to Baremetal
+Operator automated cleaning [page](../bmo/automated_cleaning.md) instead.
```

Comment:
I think these two paragraphs can be merged into one since both are referring/pointing out to same doc and on the same direction

Reply:
Would be nice, but not sure how. Because one is as prerequisite and the letter is for only BMO